ArcadiaParties-SPA/src/app/user/user-info/user-info.component.scss

```diff
@@ -0,0 +1,32 @@
+:host {
+  display: flex;
+  flex: 1 0 auto;
+  justify-content: center;
+  align-items: center;
+}
+
+.center {
+  text-align:center;
+}
+
+mat-card {
+  margin: 10% auto;
+  border-radius: 8px;
+}
+
+p {
```

Comment:
Avoid to set style to element. Use classes instead.

Reply:
It changed